capabilities/writetarget/beholder/monitor/metrics.go

```diff
@@ -0,0 +1,48 @@
+package monitor
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+
+	beholdercommon "github.com/smartcontractkit/chainlink-common/pkg/beholder"
+	"github.com/smartcontractkit/chainlink-framework/capabilities/writetarget/beholder"
+)
+
+// TODO: move to another directory / repo
+// Define a new gauge metric for account balance
+type GaugeAccBalance struct {
+	// account_balance
+	gauge metric.Float64Gauge
```

Comment:
Is float precision going to be a problem? Typically we would use minimal units like wei in order to use integers instead.

Reply:
Precision should not be a problem for balance monitoring as we don't really care about the dust but that a NOP is funded over a threshold.

The reason this is float (iirc) is because Prometheus doesn't support big numbers.